app/src/main/java/org/jellyfin/androidtv/util/Utils.java

```diff
@@ -143,8 +141,11 @@ public static boolean versionGreaterThanOrEqual(String firstVersion, String seco
     public static int getMaxBitrate() {
         String maxRate = get(UserPreferences.class).get(UserPreferences.Companion.getMaxBitrate());
         Long autoRate = get(AutoBitrate.class).getBitrate();
-        float factor = Float.parseFloat(maxRate) * 10;
-        return Math.min(autoRate != null && factor == 0 ? autoRate.intValue() : ((int) factor * 100000), 100000000);
+        if (maxRate == UserPreferences.MAX_BITRATE_AUTO && autoRate != null) {
```

Comment:
Shouldn't `.equals()` be used for the `maxRate` comparison?

Reply:
Right this is Java